src/components/MediaPicker.vue

```diff
@@ -52,66 +52,68 @@
           </div>
         </transition>
 
-        <div class="s-media-picker__controls">
-          <a
-            v-if="mediaLink"
-            class="s-media-picker__link-icon"
-            @click.stop="$emit('link-media')"
-            ><i class="icon-link"></i>
-          </a>
-
-          <transition mode="out-in" name="fade">
-            <a
-              v-if="variation === 'image' && media.selected && !mediaBroken"
-              key="media-selected-play"
-              class="s-media-picker__zoom-icon"
-              @click.stop="$emit('zoom-media')"
-              ><i class="icon-zoom"></i>
-            </a>
+        <div
+          @mouseenter="showMediaControls = true"
+          @mouseleave="showMediaControls = false"
```

Comment:
can you do like this?
@mouseenter="showMediaControls"
@mouseleave="!showMeiaControls"

Reply:
Tried it, doesn't work unfortunately. I think it's because i am not setting `showMediaControls` to a boolean value.